components/script/dom/webglprogram.rs

```diff
@@ -31,6 +31,39 @@ pub struct WebGLProgram {
     renderer: WebGLMsgSender,
 }
 
+/// ANGLE adds a `_u` prefix to variable names:
+///
+/// https://chromium.googlesource.com/angle/angle/+/855d964bd0d05f6b2cb303f625506cf53d37e94f
```

Comment:
Can you add a comment here related of how this could be done with ANGLE, and the complexities it introduces?